ffi/cpp/examples/fixeddecimal-wasm/test.cpp

```diff
@@ -0,0 +1,47 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Nit (applies to main.wasm): I think don't check in build artifacts like WASM files.  Instead just have a Makefile or some other rule.

Reply:
Oh, my bad.